src/materialized/src/bin/materialized/main.rs

```diff
@@ -119,15 +119,6 @@ struct Args {
     #[structopt(long, env = "MZ_TIMESTAMP_FREQUENCY", hidden = true, parse(try_from_str =repr::util::parse_duration), value_name = "DURATION", default_value = "1s")]
     timestamp_frequency: Duration,
 
-    /// Maximum number of source records to buffer in memory before flushing to
-    /// disk.
-    #[structopt(
```

Comment:
I personally err on the side of leaving the option for a few releases and erroring that it's no longer supported instead of it silently disappearing. Same for the `cached = true` in sql. Dunno what our policy is and if we think this is worth it pre 1.0.

Reply:
the feature was always experimental only so i feel ok ripping out -- @benesch wdyt does that sound sane?

Reply:
Yup since this was behind `--experimental` we're totally good to remove.